test/dialect/test_sqlite.py

```diff
@@ -2456,6 +2478,22 @@ def test_check_constraint(self):
                 {"sqltext": "q > 1 AND q < 6", "name": None},
             ],
         )
+        print(inspector.get_check_constraints("r"))
```

Comment:
**Federico Caselli** (CaselIT) wrote:

leftover print

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5447

Reply:
**Michael Bayer** (zzzeek) wrote:

recensionem moratus est

well better 30 seconds too late than never :)   will push now

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5447